### PR TITLE
File Added : Assignment 5,6

### PR DESCRIPTION
**Pull request by Oshin-250197**

### CODE REVIEW

**Review by SamikshaWani — APPROVED** [review]

**Review by roshnidurugwar — APPROVED** [review]

File: Oshin1/html/Assignment5/Home.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <table id="table1">
+                <tr>
+                    <td>
+                        <span class="titlet">&nbsp;Oshin Gaikwad</h3>
+                    </td>
+                </tr>
+                <tr class="bgcolor1">
+                    <td>
+                        <h2 class="right"><i>About Me&nbsp;&nbsp;</i></h2>
+                        <div class = "shadow" style="background-color:#b0c4de; background-size: cover; cover;width:96%;padding:7px;">
```
Comment (SamikshaWani) [inline Oshin1/html/Assignment5/Home.html]: add space after colan

File: Oshin1/html/Assignment6/Question1.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <body>
+        <h1>MI-11 Intern HTML Assesment</h1>
+        <hr />
+        <h2><i>Personal Details</i></h2>
+        <label for="first-name"><b>First Name:</b> </label>
+        <input type="text" name="fname" id="first-name" placeholder="First Name" />
+        <br />
+        <br />
+        <label for="last-name"><b>Last Name:</b> </label>
+        <input type="text" name="lname" id="last-name" placeholder="Last Name" />
```
Comment (SamikshaWani) [inline Oshin1/html/Assignment6/Question1.html]: placeholder second word first letter should be small

File: Oshin1/html/Assignment5/Home.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#maindiv div {
+    background-color: #8ebf42;
+    padding: 10px;
+  }
+.shadow {
+    border: 1px solid;
+    padding: 10px;
+    box-shadow: 2px 4px;
+}
+li {list-style-type:square;color: blue;font-size: 25px;border: 1px;}
```
Comment (Pawan1005) [inline Oshin1/html/Assignment5/Home.css]: make correct indentation 
add one space before {

File: Oshin1/html/Assignment5/Home.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    padding-top: 5px;
+    padding-bottom: 5px;
+    padding-right: 20px;
+}
+li{
+    font-size: 20px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+}
+#maindiv div {
```
Comment (Pawan1005) [inline Oshin1/html/Assignment5/Home.css]: where is this ID present in your file

File: Oshin1/html/Assignment5/Home.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                                    <li><a href="#resume" target="_blank">My Resume</a></li>
+                                    <li><a href="#hobbies" target="_blank">My Hobbies</a></li>
+                                    <li><a href="#photo" target="_blank">Photo Gallery</a></li>
+                                  </ul>
+                            </div>
+                        </div>
+                    </td>
+                </tr>
+            </table>
+    <body>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment5/Home.html:41]: make correct indentation

File: Oshin1/html/Assignment5/Resume.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        <link rel="stylesheet" href="resume.css" />
+    </head>
+    <body>
+        <div id="nav">
+            <h1>Resume</h1>
+        </div>
+            <div>
+                <h1>Oshin Gaikwad</h1>
+            </div>
+        </div>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment5/Resume.html:16]: remove extra div
and correct indentation

File: Oshin1/html/Assignment5/Resume.html [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+        <div id="skills" class="block">
+            <h2>Skills</h2>
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+            </div>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment5/Resume.html]: remove extra div

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**